docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -1,4 +1,112 @@
-# Grid Layout
+# Grid layout(Tizen 6.0 or later)
```

Comment:
@mijin85cho, 
I recommend removing (Tizen 6.0 or later) from title. just the note would do if we have to retain Tizen 5.5 content for Grid Layout. Also, as per our earlier discussions on retaining information for previous versions of Tizen APIs you had mentioned that we should remove old content and keep only newer contents. How to go about in this case?

Reply:
@annie-samsung 
I agree. I don't suggest to separate the page into 2 versions (Tizen 5.5 and Tizen 6.0)
Comparing to the Grid layout of Tizen 5.5, more properties are added in Tizen 6.0. 
So if  you want to keep the information that in Tizen 5.5 only `Columns` was supported, I think adding a note will do that work.

(As the Tizen 5.5's Grid layout information is merged in Tizen 6.0's Grid layout information, I think the second part of this page, from line 108 to line 132 can be removed.)

Reply:
Tizen 5.5's GridLayout document is removed.
Thank you for the reviewing.